python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -505,14 +526,15 @@ def inner(
             sensor_name = name
 
         jobs = monitored_jobs if monitored_jobs else job_selection
+        monitor_all = monitor_all_code_locations or monitor_all_repositories
```

Comment:
nit: we have this method: https://sourcegraph.com/github.com/dagster-io/dagster@143d555c5b25c63ee690632292f17e9122bc92ec/-/blob/python_modules/dagster/dagster/_utils/warnings.py?L18 which handles some edge cases automatically (e.g. what if user supplies both)

Reply:
not nit actually - i was being lazy but ty for capturing it!

Reply:
in order to capture the case where user supplies both, i converted the func signatures to accept explicitly nullable params (previously, it defaulted to False so we couldn't error here)